testing/sdk_networks.py

```diff
@@ -19,3 +27,23 @@ def check_task_network(task_name, expected_network_name="dcos"):
                     assert "name" not in network_info, \
                         "Task {task} has network name when it shouldn't has status:{status}" \
                         .format(task=task_name, status=status)
+
+
+def get_endpoints(endpoint_to_get, package_name, correct_count):
+    endpoints, _, rc = shakedown.run_dcos_command("{} endpoints {}".format(package_name, endpoint_to_get))
+    assert rc == 0, "Failed to get endpoints on overlay network"
+    endpoints = json.loads(endpoints)
+    assert len(endpoints) == correct_count, "Wrong number of endpoints, got {} should be {}" \
+        .format(len(endpoints), correct_count)
+    return endpoints
+
+
+def check_endpoints_on_overlay(endpoints):
+    ip_addresses = [e.split(":")[0] for e in endpoints["address"]]
```

Comment:
Failure scenarios in this one line. 
- endpoints might not be a mapping type
- endpoints might lack the key address
- the value at address might not be a sequence
- the elements of that value might not have the attribute split
- the attribute might not be callable
- the string might not have a colon

Not all the exceptions will be obvious